doc/errors.rst

```diff
@@ -47,13 +47,30 @@ One case that is not easily covered by the assert statement is asserting Excepti
 
 	  assert caught.exception.param == 'some_value'
 
+:func:`slash.assert_raises` will raise ``TestFailed`` exception in case the expected exception was not raised:
+
+.. code:: python
+
+	 >>> with slash.assert_raises(Exception) as caught: # doctest: +IGNORE_EXCEPTION_DETAIL
+	 ...    pass
+	 Traceback (most recent call last):
+	     ...
+	 TestFailed: ...
+
+In case the test should allow specific exception, but not enfore it (ex. timing issue), you can use :func:`slash.allow_raises`:
+
+.. code:: python
+
+	  >>> with slash.allow_raises(Exception) as caught:
```

Comment:
`allow_raising` still doesn't sound like a context, because it is not a progressive verb (it would be "allowing_raise" if we wanted this form).

Perhaps "allowing_exceptions"?